### ISSUE TITLE
Not Working On macOS Big Sur

### ISSUE THREAD

**Issue by tikkanenfelix** [Makeblock-official/Makeblock-USB-Driver#6]
I've been trying to search for an answer for days, still haven't found anything that would work on Big Sur nor windows in fact.

**Comment by jonmccon** [comment]
Yeah I tried to work around this for a while but actually ended up replacing the original electronics with a standard arduino and GRBL shield. Granted those came with their own issues, but are still actively supported.
